include/z64player.h

```diff
@@ -158,7 +159,7 @@ typedef enum {
 } PlayerDoorType;
 
 
-#define PLAYER_LIMB_BUF_COUNT PLAYER_LIMB_MAX + 2 // 2 extra entries in limb buffers?
+#define PLAYER_LIMB_BUF_COUNT ALIGN8(PLAYER_LIMB_MAX)
```

Comment:
Do we want to turn the tables into buffers now, or save that for proper documentation?